content/home/examples/a-stateful-component.md

```diff
@@ -1,7 +1,8 @@
 ---
-title: A Stateful Component
+title: Een Stateful Component
```

Comment:
Hoe vertalen we stateful? Of misschien eerder ‘een component met state’

Reply:
@duivvv so far the consensus has been to not translate "stateful". This discussion is available [here](https://github.com/reactjs/nl.reactjs.org/issues/5). Please post your relevant suggestions there as well, so we can stay consistent across documents.

Reply:
Sorry about this, shouldn’t have read it before commenting

Reply:
@duivvv Please resolve your comments in these scenarios instead of leaving them  for the reviewer.